src/catalog/src/information_schema/columns.rs

```diff
@@ -69,6 +71,26 @@ impl InformationSchemaColumns {
             self.catalog_provider.clone(),
         )
     }
+
+    pub fn to_stream(&self) -> Result<SendableRecordBatchStream> {
```

Comment:
Looks like this method is duplicated?

Reply:
Do you mean `InformationSchemaTables::to_stream()` and `InformationSchemaColumns::to_stream()`? They are two structures. We can make a general `StreamingTable` (https://github.com/GreptimeTeam/greptimedb/pull/1639#discussion_r1205001903) later